code/modules/food/food/condiment.dm

```diff
@@ -453,7 +453,10 @@
 
 //End of MRE stuff.
 
-/obj/item/weapon/reagent_containers/food/condiment/advanced/flour
+/obj/item/weapon/reagent_containers/food/condiment/carton/
+	var/do_overlay = 1
```

Comment:
Nit: `TRUE`/`FALSE`

I appreciate you taking the time to code this, though honestly looking at the changes I'm waffling and don't particularly care about the overlay thing anymore.